### ISSUE TITLE
a folder saved with an excessive long file name (named as the entire bee movie script)

### ISSUE THREAD

**Issue by Maliceowo** [Neos-Metaverse/NeosPublic#2754]
### Describe the bug?

so i was messing around with my friend yoho with a txt with the entire bee movie script in it  added the text to random stuff then shown me a file named the entire bee movie script, i was dumb and saved it to show other people, but thats where the issue comes into play, i went to try to delete it to put it in another folder and it wouldnt delete, i tried to tripple click delete and it deleted, problem is it came back after i came back online when you highlight it, it pretty much fills the entire half of the screen, asked nexulan redirected me to shifty, shifty helped me try multiple ways, nothing worked.

### Relevant issues

doesnt look like there is any relevant issues to it

### To Reproduce

pretty much just saving the file

### Expected behavior

i expected to see it fill the entire screenexpected to be able to delete it 

### Log Files

[Uploading MALICES-LAPTOP - 2021.8.2.936 - 2021-08-07 20_25_42.log…]()

### Screenshots

![un997](https://user-images.githubusercontent.com/88605190/128620300-2fea2c1e-e7be-4035-a967-84a99f838ba8.png)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2021.8.2.936

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

### Reporters

Maliceowo (Malice#0424)

**Comment by Duskitten** [comment]
Some other behaviors of this  are that, it can be deleted from your "Inventory" on the client, but reloading of the inventory of any sort causes it to respawn due to it not being deleted from the cloud, attempting to delete it from the legacy inventory also doesn't work.

**Comment by shadowpanther** [comment]
Updated the description and the reproduction steps to be more concise.

**Comment by Frooxius** [comment]
Creation of excessively long names has been fixed in 2021.9.3.1281 by @ProbablePrime. We'll probably have to delete those instances manually however, we'll look into it more soon.

**Comment by ProbablePrime** [comment]
Yup, cloud related fixes are on my list for now we've "Dam'ed the river" of this issue :)

**Comment by TheBasementNerd** [comment]
Are there any solutions to removing folders with such a name issue, such as deleting its parent folder if its under one? Or are we pretty much screwed on getting rid of it from our inventory?
